backend/api/v1beta1/update_status.proto

```diff
@@ -0,0 +1,41 @@
+// Copyright 2024.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "github.com/foundation-model-stack/fms-lm-eval-service/backend/api/v1beta1";
+
+enum UpdateStatus {
```

Comment:
I believe you know that this is for the status of this API call, which means the API call fails or not. any better naming suggestion to avoid confusion? `ResponseCode`?

Reply:
rename it to `ResponseCode`